sequencer/build.gradle

```
@@ -45,6 +45,8 @@ dependencies {
   implementation "${besuArtifactGroup}.internal:rlp"
 
   implementation 'com.google.code.gson:gson'
+  implementation 'com.github.ben-manes.caffeine:caffeine:3.1.8'
+
```

No need to specify the version since we inherit it from the Besu BOM

```suggestion
  implementation 'com.github.ben-manes.caffeine:caffeine'
  
  implementation 'com.google.code.gson:gson'
```